README.md

```diff
@@ -1,31 +1,59 @@
 # Personality Paragrapher
 
-Personality Paragrapher is an application that defines personality traits based off of music lyrics passed in. A user can create an account, search for a song, and recieve a response of the personality detected from the song lyrics.
+Personality Paragrapher is an application that defines personality traits of a playlist created by a user. The user must create an account and add to a playlist by typing in the artist and song title. When the user performs a `GET` request for their playlist personality, it will return an object with percentages of how the playlist matched on 5 characteristics.
 
 ---
 
 ## Installing and Getting Started
 
+### Installing
 Fork and git clone this repository to your local computer, navigate to the root directory of the repository and run `npm i` in the command line, this will install all necessary packages.
 
-<!-- More about how to use -->
+### Getting Started
 
+To sign up:
+
+```http
+http://<>/signup username=<username> password=<password> email=<email>
+```
+
+To sign in:
+```http
+http://<>/signin username=<username> password=<password>
 ```
-code block for http examples
+
+To add music to your playlist:
+```http
+http://<>/song artist=<artist name> title=<song title>
+```
+
+To get playlist personality:
+```http
+http://<>/personality userId=<userId>
```

Comment:
I could be wrong on this and if I am then I'm overthinking it but when I envisioned this request I was thinking we would be sending a GET to /api/v1/personality and providing the bearer token since that requires that the user be logged/signed in similar to how in the previous lab we wouldn't let anyone access the /api/v1/photo endpoints without sending a bearer token. When the bearer token middleware processes the request, it will add the 'user' object onto the request object at which point we can handle the remaining steps. What do you think?

Reply:
I only added some code blocks as placeholders really, I'm happy to change these once we know exactly what the requests are. If you don't mind merging this progress, then I'll continue to modify tomorrow.

Reply:
Sounds good!